docs/Docker_Instructions.md

```diff
@@ -0,0 +1,48 @@
+# Running Dockerized ConnectedVCS Tools
+If you are running ConnectedVCS Tools using a docker image, you can use the following instructions:
+
+## Prerequisites
+ConnectedVCS Tools has been developed using Ubuntu 20.04 and Ubuntu 22.04. Further testing with other operating systems is needed before guidance is created. For the moment, please use Ubuntu 20.04 or later [Ubuntu LTS Release](https://releases.ubuntu.com/).
+
+### Install Docker CE
+Instructions for installing Docker may change, so please use the current instructions at the Docker website:
+https://docs.docker.com/desktop/install/linux-install/.
+
+## Run the ConnectedVCS Tools Image
+
+### Build a Custom Image
+
+1. Clone the ConnectedVCS Tools respository:
+```
+git clone https://github.com/usdot-fhwa-stol/connectedvcs-tools.git
+```
+2. Create a new Bing Maps API Key (Basic, Website) using https://www.bingmapsportal.com/.
```

Comment:
It seems like Bing Map API keys are deprecated and new users will not be able to create these keys. Please see the above comment.